### DISCUSSION TITLE
Welcome to Recipes discussions!

### DISCUSSION THREAD

**Discussion by erika-shorten** [weaviate/recipes#62] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Josephrp** [comment]
hey there folks 🙋🏻‍♂️ just a builder from paris with a taste for cool demos, and everybody knows @erika-cardenas is making a lot of those recently, that's how i heard of this place and why i'm going to follow along :-) looking forward to more of the same 🚀

**Reply by erika-shorten** [reply]
Welcome @Josephrp! 🥳 

Thanks so much for following along, and feel free to contribute notebooks as you see fit!
